pkg/services/ngalert/provisioning/provenance.go

```diff
@@ -0,0 +1,14 @@
+package provisioning
+
+import (
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+func canUpdateWithProvenance(storedProvenance, provenance models.Provenance, errFun func() error) error {
```

Comment:
Comment on how the function behaves. It's not apparent to the caller if it isn't a `bool` return type.